x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -146,7 +149,7 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
   const [shouldBulkCloseAlert, setShouldBulkCloseAlert] = useState(false);
   const [shouldDisableBulkClose, setShouldDisableBulkClose] = useState(false);
```

Comment:
I see this component accepts `ruleName` as prop what will be the value of this prop in case of new List'Exception not linked to a rule?
https://github.com/elastic/kibana/blob/967629582f014ca804e1555578237b2b8156b735/x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx#L132

Reply:
This component is going to be almost entirely rewritten once the new components are in. It will no longer rely on rule name to fill out any data in the flyout.